From 011d2961ffcee2581e9a347d02aab0e656e1acd8 Mon Sep 17 00:00:00 2001
From: Mathieu Westphal <mathieu.westphal@kitware.com>
Date: Sun, 8 Jan 2023 15:04:23 +0100
Subject: [PATCH] Adding INSTALL_RPATH

---
 adm/cmake/occt_toolkit.cmake | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/adm/cmake/occt_toolkit.cmake b/adm/cmake/occt_toolkit.cmake
index 517d1b47f..854b98cca 100644
--- a/adm/cmake/occt_toolkit.cmake
+++ b/adm/cmake/occt_toolkit.cmake
@@ -318,6 +318,13 @@ if (CURRENT_MODULE)
       set_target_properties (${PROJECT_NAME} PROPERTIES BUILD_WITH_INSTALL_RPATH 1 INSTALL_NAME_DIR "${INSTALL_NAME_DIR}")
     endif()
   endif()
+
+  if (APPLE)
+   set_target_properties(${PROJECT_NAME} PROPERTIES INSTALL_RPATH @loader_path)
+  elseif (UNIX)
+    set_target_properties(${PROJECT_NAME} PROPERTIES INSTALL_RPATH $ORIGIN)
+  endif ()
+
 endif()
 
 get_property (OCC_VERSION_MAJOR GLOBAL PROPERTY OCC_VERSION_MAJOR)
-- 
2.39.0

