diff --git a/cmake/draco_build_definitions.cmake b/cmake/draco_build_definitions.cmake
index b7e6be4..05ec6c2 100644
--- a/cmake/draco_build_definitions.cmake
+++ b/cmake/draco_build_definitions.cmake
@@ -32,9 +32,7 @@ macro(set_draco_target)
     set(draco_plugin_dependency draco_static)
   endif()
 
-  if(BUILD_SHARED_LIBS)
-    set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-  endif()
+  set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 endmacro()
 
 # Configures flags and sets build system globals.
