diff --git a/gdal.cmake b/gdal.cmake
index ab1c82f2c0..3c74b0074c 100644
--- a/gdal.cmake
+++ b/gdal.cmake
@@ -311,6 +311,7 @@ else ()
           message(WARNING "CMAKE_INSTALL_RPATH=${CMAKE_INSTALL_RPATH} will be ignored and replaced with ${base};${base}/${relDir} due to GDAL_SET_INSTALL_RELATIVE_RPATH being set")
       endif()
       set(CMAKE_INSTALL_RPATH ${base} ${base}/${relDir})
+      set_target_properties(${GDAL_LIB_TARGET_NAME} PROPERTIES INSTALL_RPATH ${base})
   endif()
 endif ()
 
